CMakeLists.txt

```diff
@@ -5,9 +5,13 @@ if(NOT WIN32)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
 endif()
 
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
-#find_package(geographic_msgs REQUIRED)
+find_package(geographic_msgs REQUIRED)
```

Comment:
I don't think it's released yet though. I saw the comment in the issue. Right now it needs to be cloned manually and activate that workspace. This is kind of important as well https://github.com/ros-geographic-info/geographic_info/pull/26 which hasn't been approved yet :(

Reply:
uh, then we cant merge this @klintan, it'll not build in CI or on releasing debians

Reply:
No worries, I'll try to push to get the geographic_msgs released, maybe we can keep this PR open until then (hopefully won't be months) and as soon as it's there, this will be mergeable.

Reply:
*sigh* ok, now, weeks later, I have lobbied all the parties required to merge the ROS2 port PR, it will be released in the next dashing sync assuming I didnt mess it up. But now I have a new shiny GitHub organization on my profile, so not a total loss of time :-)  
https://github.com/ros/rosdistro/pull/22324